secator/runners/python.py

```
@@ -0,0 +1,123 @@
+		# Call super().__init__
+		super().__init__(
+			config=config,
+			inputs=inputs,
+			results=results,
+			run_opts=run_opts,
+			hooks=hooks,
+			validators=validators,
+			context=context)
+
+	@staticmethod
+	def _validate_input_nonempty(self, inputs):
```

Ruff (0.14.7)

Unused static method argument: `self`

(ARG004)